random-dataset/README.md

```diff
@@ -0,0 +1 @@
+This folder contains random data for testing
```

Comment:
I would keep this folder for future PRs, probably in `test/resources/data` directory.

Reply:
perfect